cellfinder_napari/tests/test_cellfinder_napari.py

```diff
@@ -7,51 +7,33 @@
 from PIL import Image
 
 from cellfinder_napari.detect import detect
-
-test_data_dir = pathlib.Path(__file__) / ".." / ".." / ".." / "test_data"
-
-
-def load_stack(directory: pathlib.Path) -> np.ndarray:
-    """
-    Load a stack of .tif image files in the given directory.
-    """
-    tif_glob = str((directory / "*.tif").resolve())
-    ims = []
-    for f in sorted(glob.glob(tif_glob)):
-        ims.append(np.array(Image.open(f)))
-    return np.stack(ims, axis=0)
-
-
-# @pytest.fixture
-# def signal_image():
-#    return load_stack(test_data_dir / "crop_planes" / "ch0")
-
-
-# @pytest.fixture
-# def background_image():
-#    return load_stack(test_data_dir / "crop_planes" / "ch1")
-
-
-def test_detect_function(
-    make_napari_viewer,
-):  # , signal_image, background_image):
-    # Smoke test that running cell detection from the widget works
-
-    # Create viewer and add images
+from cellfinder_napari.detect.input_containers import (
+    ClassificationInputs,
+    DataInputs,
+    DetectionInputs,
+    MiscInputs,
+)
+from cellfinder_napari.detect.thread_worker import Worker
+from cellfinder_napari.sample_data import load_sample
+
+
+def test_add_detect_widget(make_napari_viewer):
+    # Smoke test to check that adding detection widget works
```

Comment:
I might be wrong, but it looks like this is a description of the test and, as such, should be a docstring
```suggestion
    """
    Smoke test to check that adding detection widget works
    """
```